prebuild.sh

```diff
@@ -0,0 +1,8 @@
+# This is a HACKY script to fix build errors that occur when using ethereumjs v6+
+# Please refer to this issue if you don't understand what this is all about:
+# https://github.com/ethereumjs/ethereumjs-monorepo/issues/2295#issue-1377817460
+# Hopefully this will be removed when the issue is properly resolved.
```

Comment:
nit: Is there already an upcoming release branch mentioned [here](https://github.com/ethereumjs/ethereumjs-monorepo/issues/2295#issuecomment-1278336434) we could reference here to keep an eye on?

Reply:
So the fix had already been merged to their `master` branch. See [here](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2303).
And from the last comment in the PR, it looks like this will get integrated into `release` immediately, so I'd say we wait for them to integrate it and then I'll push another commit to this branch that will delete the `prebuild.sh` script.